### DISCUSSION TITLE
Is there an equivalent SDK for the "fga model get" command?

### DISCUSSION THREAD

**Discussion by keitherw** [openfga/community#381] [General]
Looking to see if there is an SDK for golang to obtain the model structure. I want to be able to obtain the conditions to validate incoming JSON values.

**Comment by aaguiarz** [comment]
`fga model get` calls ReadAuthorizationModels and gets the first one (which is the latest one that was written). Would that work?

**Reply by keitherw** [reply]
I did the following but not sure how to read "model"

	storeId, _ := fgaClient.GetStoreId()
	options := client.ClientReadAuthorizationModelsOptions{
		PageSize: openfga.PtrInt32(10),
		// You can rely on the store id set in the configuration or override it for this specific request
		StoreId: &storeId,
	}
	
	model, err := fgaClient.ReadAuthorizationModels(context.Background()).Options(options).Execute()
	if err != nil {
		fmt.Println("Error getting model:", err)
		return
	}

**Reply by keitherw** [reply]
doesn't look like it returns the conditions

**Comment by aaguiarz** [comment]
It should. Does the CLI return the conditions?

The Go SDK has a method to get the latest model too https://github.com/openfga/go-sdk/blob/62a6bbc19180744b9221c029b2711e5e4ca90312/example/example1/example1.go#L87

**Reply by keitherw** [reply]
Thanks, I am able to get my conditions with this

conditions := (*latestAuthorizationModel.AuthorizationModel).GetConditions()
